File: java/src/backjoon/backjoon_1_10.java
package backjoon;

import java.util.Scanner;

public class backjoon_1_10 {


    public static void main(String[] args) {

        // 입력형식
        Scanner sc = new Scanner(System.in);

        // ex : 3 5 6 2 4
        // 첫 번째 행에 공백(space)을 구분자로 숫자가 주어진다
        // 각 숫자는 한 자리 숫자로 주어진다 (0과 같거나 크고, 10보다 작은 숫자)
        // 같은 숫자가 중복되어 나타나지 않는다
        String input = sc.nextLine();
        String[] splitInput = input.split(" ");
        int[] arr = new int[splitInput.length];

        int count = 0;
        for(String value : splitInput) {
            arr[count++] = Integer.valueOf(value);
        }

        int sum = 0;
        int sub = 0;
        int multi = 0;
        int div = 0;
        int remain = 0;

        sum = arr[0] + arr[1];
        sub = arr[0] - arr[1];
        multi = arr[0] * arr[1];
        div = arr[0] / arr[1];
        remain = arr[0] % arr[1];

        System.out.println(sum);
        System.out.println(sub);
        System.out.println(multi);
        System.out.println(div);
        System.out.println(remain);

    }

}

File: kotlin/src/backjoon/kotlin/backjoon_1_10.kt
package backjoon.kotlin


fun main(args: Array<String>) {

//    var sc: Scanner = Scanner(System.`in`)
//    var book_kotlin_200.input = sc.nextLine()

    // A+B, A-B, A*B, A/B(몫), A%B(나머지)를 출력하는 프로그램을 작성하시오.

    var sc:String? = readLine()
    var input:String = sc.toString()

    var splitInput = input.split(" ")
    var arr = Array<Int>(100,{0})

    var count:Int = 0
    for(value in splitInput) {
        arr[count++] = value.toInt()
    }

    var sum:Int = 0
    var sub:Int = 0
    var multi:Int = 0
    var div:Int = 0
    var remain:Int = 0

    sum = arr[0] + arr[1]
    sub = arr[0] - arr[1]
    multi = arr[0] * arr[1]
    div = arr[0] / arr[1]
    remain = arr[0] % arr[1]

    println(sum)
    println(sub)
    println(multi)
    println(div)
    println(remain)

}